WordPress/src/main/java/org/wordpress/android/ui/main/ActionListItemViewHolder.kt

```diff
@@ -15,11 +16,24 @@ class ActionListItemViewHolder(
         val actionIcon: ImageView = this.itemView.findViewById(R.id.action_icon)
         val actionTitle: TextView = this.itemView.findViewById(R.id.action_title)
 
-        actionIcon.setImageResource(action.iconRes)
-        actionTitle.setText(action.labelRes)
+        if (action.iconRes > 0) {
+            actionIcon.setImageResource(action.iconRes)
```

Comment:
This isn't exactly part of this PR, so feel free to ignore this comment.

We try to load all images, both local and remote, using `ImageManager` -> `imageManager.load(actionIcon, action.iconRes)`. It uses glide underneath and supports various types of optimisations/caches.

Reply:
Thanks for sharing. Changed here 7ca16a944a46c9731a804b19c7e969e3ca239e3f